README.md

```diff
@@ -1 +1,19 @@
-# Fill me in with information
+Official MongoDB Node.js Driver Quick Start
+===========================================
```

Comment:
Comment: Should this use a markdown header? It seems to render, however I'm not sure I've seen this syntax used in Markdown before. Applies below as well if you feel this is applicable.

```suggestion
# Official MongoDB Node.js Driver Quick Start
```

Reply:
It's valid markdown syntax, not sure either way is better than the other. Is there any value in changing it?

Reply:
Just dug into it further and [yup this seems to be a valid alternate heading syntax](https://www.markdownguide.org/basic-syntax/#alternate-syntax). Sorry for the confusion. All good from me.